openadapt/config.py

```diff
@@ -409,7 +411,34 @@ def print_config() -> None:
                 )
```

Comment:
Regarding the placement of this logic, I think it should be moved out of `config.py` and into e.g. `errors.py` or similar.

IIRC the purpose of `ERROR_REPORTING_BRANCH` was to avoid reporting errors on branches that were in development. I think that still makes sense. What do you think?

Reply:
Tracked in https://github.com/OpenAdaptAI/OpenAdapt/issues/687